packages/amplify_datastore/CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+## 0.2.0 (2021-06-30)
+
+### Features
```

Comment:
```suggestion
## 0.2.0 (2021-06-30)

### Breaking Changes

- The previously deprecated configureModelProvider function has been removed.

### Features
```

Reply:
We should include the breaking change that @cshfang mentioned.